Bottom: 32b1ced78f7bb1e9a0fc001c61977cf27d839d0d
Top:    c2d06a1e4fe5f647305190858ac6ff310dc1edd5
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-12-31 23:52:01 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index 0272133..81e28d1 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -306,7 +306,7 @@ define KernelPackage/sound-soc-rt5350-le89156
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-le89156.ko \
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-rt5350-le89156.ko
   AUTOLOAD:=$(call AutoLoad,57,snd-soc-rt5350-le89156)
-  DEPENDS:=@TARGET_ramips +kmod-snd-soc-rt5350
+  DEPENDS:=@TARGET_ramips +kmod-sound-soc-rt5350
   $(call AddDepends/sound)
 endef
